src/flaskwebgui.py

```diff
@@ -207,6 +211,24 @@ def __post_init__(self):
             print("path to chrome not found")
             self.browser_command = [PY, "-m", "webbrowser", "-n", self.url]
 
+    
+    def prevent_resizing(self):
+        print("\033[31mSCREEN RESIZING FUTURE ONLY WORK IN WINDOWS SYSTEM!\033[0m")

```

Comment:
If you find a way to prevent window resizing for all platforms Windows/Linux/Mac then this feature would be helpful for some cases.